### DISCUSSION TITLE
WebSocket connection to fail

### DISCUSSION THREAD

**Discussion by raghu-gade** [socketio/socket.io#5252] [Q&A]
Here is my socket connection code. It's working fine with socket URL 'wss://my-application.eastus.cloudapp.azure.com' in application we need to add ingress path 'nodeserver'. it is taken and connected initially and disconnected immediately next subsequent connections fail error showing.
**WebSocket connection to 'wss://my-application.eastus.cloudapp.azure.com/nodeserver/socket.io/?EIO=4&transport=websocket' failed:** 

```
this.socket = io('wss://my-application.eastus.cloudapp.azure.com',
    {
        transports: ["websocket"],
        path: "/nodeserver/socket.io/",
        auth: {
          userId: userId,
          token: token
        }
      });
 this.socket.on('connect_err', err => {
      console.log(err);
    });
 this.socket.on("disconnect", (reason) => {
      console.warn("Disconnected:", reason);
    }
  });
```
  While disconnect the socket showing message **Disconnected: transport close**
<img width="702" alt="bug1" src="https://github.com/user-attachments/assets/cf9d9023-2600-4ab6-8be8-34073b4748c1">
<img width="638" alt="bug2" src="https://github.com/user-attachments/assets/5f0124a7-d71c-468b-ac15-34293bc85fcf">
Any alternate solution with ingress path socket connection

**Comment by darrachequesne** [comment]
That's weird, according to your screenshot the client actually reaches the Socket.IO server, since we see the handshake packets.

If I understand correctly, it works well without ingress path?

**Reply by raghu-gade** [reply]
It's working fine without the ingress path. However, adding the ingress path is necessary due to the application's requirements. Any idea what might be causing the issue with the path configuration?
